src/main/java/no/nav/foreldrepenger/oversikt/tilgangskontroll/AdresseBeskyttelse.java

```diff
@@ -0,0 +1,16 @@
+package no.nav.foreldrepenger.oversikt.tilgangskontroll;
+
+import java.util.Set;
+
+import no.nav.pdl.AdressebeskyttelseGradering;
+
+public record AdresseBeskyttelse(Set<AdressebeskyttelseGradering> gradering) {
+
```

Comment:
Se kommentar i PdlKlientSystem. 

Forøvrig
public record AdresseBeskyttelse(Set<AdressebeskyttelseGradering> gradering) {
   public AdresseBeskyttelse {
        Objects.requreNonNull(gradering, "nullable sets lukter litt antipattern");
   }

Reply:
Kan ikke være null. Var en unødvendig safeguard. Men kan være empty så vidt jeg vet.